extensions/objects/templates/svg-filename-editor.component.spec.ts

```diff
@@ -109,6 +122,9 @@ describe('SvgFilenameEditor', function() {
       deferred.resolve('sample-csrf-token');
       return deferred.promise;
     });
+    // @ts-ignore inorder to ignore other Image object properties that
```

Comment:
Why do we have this comment? I don't understand...

Reply:
That comment explains the reason why a ts-ignore is used. The mock image object does not contain all the properties of image so ts leaves an error. So in order to ignore that the @ ts-ignore is used.